src/app/oss.module.ts

```diff
@@ -0,0 +1,481 @@
+        ProfileComponent,
+        PurgeVaultComponent,
+        RecoverDeleteComponent,
+        RecoverTwoFactorComponent,
+        RegisterComponent,
+        ReusedPasswordsReportComponent,
+        SearchCiphersPipe,
+        SearchPipe,
+        SelectCopyDirective,
+        SendAddEditComponent,
+        SendComponent,
+        SendInfoComponent,
+        SettingsComponent,
+        ShareComponent,
+        SsoComponent,
+        StopClickDirective,
+        StopPropDirective,
+        TaxInfoComponent,
+        ToolsComponent,
+        TrueFalseValueDirective,
+        TwoFactorAuthenticatorComponent,
+        TwoFactorComponent,
+        TwoFactorDuoComponent,
+        TwoFactorEmailComponent,
+        TwoFactorOptionsComponent,
+        TwoFactorRecoveryComponent,
+        TwoFactorSetupComponent,
+        TwoFactorVerifyComponent,
+        TwoFactorWebAuthnComponent,
+        TwoFactorYubiKeyComponent,
+        UnsecuredWebsitesReportComponent,
+        UpdateKeyComponent,
+        UpdateLicenseComponent,
+        UserBillingComponent,
+        UserLayoutComponent,
+        UserSubscriptionComponent,
+        VaultComponent,
+        VerifyEmailComponent,
+        VerifyEmailTokenComponent,
+        VerifyRecoverDeleteComponent,
+        WeakPasswordsReportComponent,
+    ],
+    entryComponents: [
+        AddEditComponent,
+        ApiKeyComponent,
+        AttachmentsComponent,
+        BulkActionsComponent,
+        BulkDeleteComponent,
+        BulkMoveComponent,
+        BulkRestoreComponent,
+        BulkShareComponent,
+        CollectionsComponent,
+        DeauthorizeSessionsComponent,
+        DeleteAccountComponent,
+        DeleteOrganizationComponent,
+        EmergencyAccessAddEditComponent,
+        EmergencyAccessAttachmentsComponent,
+        EmergencyAccessConfirmComponent,
+        EmergencyAccessTakeoverComponent,
+        EmergencyAddEditComponent,
+        FolderAddEditComponent,
+        ModalComponent,
+        OrgAddEditComponent,
+        OrgAttachmentsComponent,
+        OrgBulkStatusComponent,
+        OrgBulkConfirmComponent,
+        OrgBulkRemoveComponent,
+        OrgCollectionAddEditComponent,
+        OrgCollectionsComponent,
+        OrgEntityEventsComponent,
+        OrgEntityUsersComponent,
+        OrgGroupAddEditComponent,
+        OrgPolicyEditComponent,
+        OrgResetPasswordComponent,
+        OrgUserAddEditComponent,
+        OrgUserConfirmComponent,
+        OrgUserGroupsComponent,
+        PasswordGeneratorHistoryComponent,
+        PurgeVaultComponent,
+        SendAddEditComponent,
+        ShareComponent,
+        TwoFactorAuthenticatorComponent,
+        TwoFactorDuoComponent,
+        TwoFactorEmailComponent,
+        TwoFactorOptionsComponent,
+        TwoFactorRecoveryComponent,
+        TwoFactorWebAuthnComponent,
+        TwoFactorYubiKeyComponent,
+        UpdateKeyComponent,
+    ],
+    providers: [DatePipe, SearchPipe],
+    bootstrap: [],
+})
+export class OSSModule { }
```

Comment:
Naming nit, similar to SSO as `Sso`, please use pure PascalCase for class names, `OssModule`